wp-api.js

```diff
@@ -179,7 +179,7 @@
 				// Add any non empty args, merging them into the args object.
```

Comment:
We shouldn't change the JS files here, they should be changed in the [JS repo, found here](https://github.com/WP-API/client-js).  We need to remove these from here for now and when they are put into the JS repo we can build them and remerge them into this repo.